ui/packages/atlasmap/src/Atlasmap/dialogs/useParametersDialog.tsx

```diff
@@ -20,6 +20,100 @@ import { useToggle } from '../../Atlasmap/utils';
 
 type ParametersCallback = (parameters: IParameter[]) => void;
 
+export function getCSVParameterOptions(): IParameter[] {
```

Comment:
Could this be moved into `contracts/` as an interface, e.g. `ICSVParameterOptions`? maybe IParameter as well.

Reply:
It might have to be a class to have values, possibly with `static readonly` - https://stackoverflow.com/questions/37265275/how-to-implement-class-constants

Reply:
And ideally these parameter list should be retrieved from backend as the original source is the backend CSV module. But it would need a bigger surgery to establish a procedure to retrieve inspection parameters beforehand.

Reply:
@igarashitm - do you see this as an addition to the atlasmap-core contracts or as  a new folder atlasmap/src/contracts?

Reply:
Actually it probably makes more sense in atlasmap-core/src/contracts since we could use them later if/when we retrieve the inspection params from the backend

Reply:
The existing `@atlasmap/core:/contracts/documents/csv.ts` I would say, in principle @atlasmap/atlasmap shouldn't declare any contracts but just use the one from @atlasmap/core. We'll even migrate existing (#2975, #2883).

Reply:
Made stories run the Parameters dialog for both select and edit.
Established ICsvParameterOptions static class in contracts.